src/lib/dom-api.html

```diff
@@ -593,6 +593,55 @@
 
     };
 
+    Object.defineProperties(DomApi.prototype, {
+      activeElement: {
```

Comment:
Let's separate the implementations of Shadow/Shady. Should make this more understandable.